src/libs/conduit/conduit_node.hpp

```diff
@@ -3708,10 +3708,18 @@ class CONDUIT_API Node
     /// info() creates a node that contains metadata about the current
     /// node's memory properties
     void             info(Node &nres) const;
+
     /// TODO: this is inefficient w/o move semantics, but is very
     /// convenient for testing and example programs.
     Node             info() const;
 
+    ///
+    /// describe() creates a node that replaces each leaf with
+    ///  descriptive statistics (count, mean, min, max) and a string repd 
+    ///  values summary
+    void             describe(Node &nres) const;
+    void             describe(const Node &opts, Node &nres) const;
```

Comment:
Could we also add `to_summary_string` and `to_summary_string_default` to the `conduit::Node` type? This would make it much easier to print summary information from debugging tools like `gdb`.

Reply:
This is an awesome idea, I will address in a new PR, here is an issue associated with this 

https://github.com/LLNL/conduit/issues/674